src/controllers/courseController.ts

```diff
@@ -57,7 +58,8 @@ export class CourseController {
 		logger.debug('Course Overview page')
 
 		return async (request: Request, response: Response) => {
-			response.render('page/course/course-overview')
+			const faceToFaceModules = response.locals.course.modules.filter( (module: Module) => module.type == 'face-to-face' );
```

Comment:
MAINTENANCE - This is a comment for the whole team. `reponse.locals` a stateful variable is a big no! If you need to store and fetch the data then encapsulate it in a `service` or `store` of some sort.

MAINTENANCE - `face-to-face` should be a constant

Reply:
`response.locals` - I understand the team agreed to use it for making data available to templates until further notice

`face-to-face` - the context is checking equality to `module.type` which is of `String Literal Type` type, in that case it is effectively a constant, since the compiler will not allow anything that's not a member of the type.

Reply:
I am not talking about compiler definitions. If I were to update `face-to-face` to `classroom` in the future, do you expect the developer to string search the whole project?

`response.locals` I will circulate around an email now.